controllers/OrganisationsController.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('./BaseController');
+const { orgLookup } = require('../services/OrganisationSearchService');
+
+class OrganisationsController extends BaseController {
```

Comment:
Could we structure these into a subfolder for "UserManagement"? Just so it's a bit more obvious that this is not really related to the primary function of LEV

Reply:
UserManagement/ directory added in Controllers/ and Services/ and move relevant files.